python/jobs/beam_sample_job.py

```diff
@@ -1,8 +1,5 @@
 """!
```

Comment:
I just realized that this job is basically just repeating the BeamCoords() and RandomSample() part of beam_gen. I don't think we need to keep the example and this script.